### ISSUE TITLE
Compiled JS file is missing closure_wrapper function

### ISSUE THREAD

**Issue by chanks** [rustwasm/wasm-bindgen#1563]
### Describe the Bug

wasm-bindgen doesn't seem to be generating a necessary `__wbindgen_closure_wrapper77` function in the .js file it emits. Things were working for me up until I started using Dodrio in my crate - the one-line change that causes this bug to appear is adding `let vdom = dodrio::Vdom::new(&app_container, app);`.

### Steps to Reproduce

I'm working on extracting a small reproduction, but figured I'd open this up early in case the cause is obvious to anyone. I'm calling wasm-pack like `wasm-pack build client --dev -t web`.

### Expected Behavior

Expected the wasm file to load in the browser successfully.

### Actual Behavior

WebAssembly.instantiateStreaming throws:
```
`WebAssembly.instantiateStreaming` failed. Assuming this is because your server does not serve wasm with `application/wasm` MIME type. Falling back to `WebAssembly.instantiate` which is slower. Original error:
 LinkError: WebAssembly Instantiation: Import #23 module="./client" function="__wbindgen_closure_wrapper77" error: function import requires a callable

t.function.WebAssembly.instantiateStreaming.WebAssembly.instantiateStreaming.catch.e | @ | client.js:650
```

### Additional Context

```
wasm-pack 0.8.1
rustc 1.36.0-nightly (50a0defd5 2019-05-21)
Chromium Version 72.0.3626.121 (Official Build) Built on Ubuntu , running on LinuxMint 19 (64-bit)

wasm-bindgen = "0.2.45"
wasm-bindgen-test = "0.2.45"
wasm-bindgen-futures = "0.3.22"
```

I'll keep working on a full reproduction, but if there's any info I can give that will help short of that, please let me know. Thanks!

**Comment by chanks** [comment]
Ah, alright, I think I figured this out. Turns out that the `__wbindgen_closure_wrapper77` function was being generated by `wasm-pack --release` but not `wasm-pack --dev`, and a script I wrote calls `wasm-pack --dev` but then also calls `npm run build` to compile some typescript, but I'm also using the wasm-pack webpack plugin, so then the crate was being compiled a second time in release mode and I wound up using the `--dev` javascript files against the `--release` wasm file. :grimacing: 

I suppose chalk this one up to my trying to hack together a build pipeline to combine wasm-pack with typescript. Anyway, I can figure it out from here, sorry for the noise.
